index.js

```diff
@@ -84,12 +84,12 @@ function updateSpecFromPackage() {
     spec.info.license = { name: packageInfo.license };
   }
 
-  spec.info.description = '[Specification JSON](' + packageInfo.baseUrlPath + '/api-spec)';
-  if (packageInfo.baseUrlPath) {
+  packageInfo.baseUrlPath = packageInfo.baseUrlPath || '';
+  const baseSpecPath = packageInfo.baseUrlPath + '/api-spec';
+  spec.info.description = `Specification JSONs: [v2](${baseSpecPath}/v2), [v3](${baseSpecPath}/v3).`;
+  if (packageInfo.baseUrlPath !== '') {
     spec.basePath = packageInfo.baseUrlPath;
```

Comment:
Does empty `spec.basePath` affect swagger-ui tests execution?

Reply:
it doesn't just wanted to avoid condition in line 88